README.md

```diff
@@ -58,10 +58,30 @@ Result: qOfcZig AOUbig
 With Offset: 51312
 ```
 
+## Tests
+First, the directory of the location of caesarlib must be selected.
+```sh
+cd caesarlib.rs
```

Comment:
I don't think this is necessary.

I guess — Enter the caesarlib repository and run: ``cargo run --verbose`` — is enough.